packages/react-core/src/components/ApplicationLauncher/examples/ApplicationLauncherMenu.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { ApplicationLauncher, ApplicationLauncherItem } from '@patternfly/react-core';
+
+const appLauncherItems: React.ReactElement[] = [
+  <ApplicationLauncherItem key="application_1a" href="#">
+    Application 1 (anchor link)
+  </ApplicationLauncherItem>,
+  <ApplicationLauncherItem key="application_2a" component="button" onClick={() => alert('Clicked item 2')}>
+    Application 2 (button with onClick)
+  </ApplicationLauncherItem>,
+  <ApplicationLauncherItem key="disabled_application_4a" isDisabled>
+    Unavailable application
+  </ApplicationLauncherItem>
+];
+
+export const ApplicationLauncherMenu: React.FunctionComponent = () => {
```

Comment:
Based on the conversation regarding naming, I'm wondering if this name should instead revert back to ApplicationLauncherDocumentBody, or if that does stray into "too long" territory.